recipe/build_base.sh

```diff
@@ -9,14 +9,11 @@ tests = False
 toolkit_tests = False
 sample_data = False
 
-[libs]
-system_freetype = True
-
 EOF
 
 cat conda_mpl_config.cfg
-sed -i.bak "s|/usr/local|${PREFIX}|" setupext.py
 
 export MPLSETUPCFG=conda_mpl_config.cfg
```

Comment:
Same here.

Reply:
Dropped in commit: https://github.com/conda-forge/matplotlib-feedstock/pull/373/commits/8c8807577b1ec9784120069921b097eea35b6a36